### PR TITLE
feat: Open item with

### LINKED ISSUE #776: Open with option
The app looks very good but it lacks an important feature in that I would like to choose what program a particular file opens with. Is this possible in the app? I may just not have recognised how to use this feature.

### PR DESCRIPTION
**Pull request by FelixZacher**
Adds the ""Open with"-option (see #776).

files_openWith
files_applicationPicker

TODO:

- [x]  ~~Icon?~~ Fixed with 3dd3b8f
- [x] ~~I added the string `x:Uid="BaseLayoutItemContextFlyoutOpenItemWith"`, but i'm not familiar with the translation tool, so some help with this, would be nice.~~ Fixed with 532f694

fix #776

### CODE REVIEW

**Comment by xPoppyx** [comment]
icon proposal:
Open
Open With
Captura de pantalla (2)

File: Files/UserControls/LayoutModes/GridViewBrowser.xaml
Comment (xPoppyx) [inline Files/UserControls/LayoutModes/GridViewBrowser.xaml:176]: can you add an id for this too?

**Comment by lampenlampen** [comment]
Can someone explain me, what `x:Load` does? I copied it from the `open`-item, because they are very similar.

**Comment by xPoppyx** [comment]
check my PR in your repository

**Comment by FelixZacher** [comment]
I wait for approval by @duke7553 or @yair100 for changing the icons.

**Comment by duke7553** [comment]
@lampenlampen Go for it. The current icon doesn't seem appropriate, anyways

**Comment by yaichenbaum** [comment]
The current icon for "Open" should be the icon for open with.

**Comment by FelixZacher** [comment]
@yair100 It now looks like this:
> 

You mean the other way round? or ?

**Comment by yaichenbaum** [comment]
@lampenlampen That looks great!

**Comment by lampenlampen** [comment]
Then i think we are ready to merge, if there are no complains anymore. 👍

**Review by yair100 — APPROVED** [review]

**Review by yair100 — APPROVED** [review]

**Pull request merged**
